hw/ip/otbn/doc/_index.md

```diff
@@ -372,14 +372,38 @@ The number of entries in the loop stack is implementation-dependent.
 
 ## Design Details
 
-<div class="bd-callout bd-callout-warning">
-  <h5>Note</h5>
+### Error Handling and Reporting
 
-  To be filled in as we create the implementation.
-</div>
+By design, OTBN is a simple processor and provides no error handling support to code that runs on it.
+Any errors will cause OTBN to stop executing and report the failure to the host processor.
+
+Whenever OTBN observes an error, it will generate an alert.
+This gets sent to the alert manager.
+The alert will either be fatal or recoverable, depending on the class of error: see [Alerts]({{< relref "hw/ip/otbn/doc#alerts" >}}) and [Error Conditions]({{< relref "hw/ip/otbn/doc#error-conditions" >}}) below for details.
+
+If OTBN was running when the alert occurred (this is true whenever {{< regref "STATUS.busy" >}} is high), it will also:
+- Immediately stop fetching and executing instructions.
+- Set {{< regref "INTR_STATE.done" >}} and clear {{< regref "STATUS.busy" >}}, marking the operation as completed.
```

Comment:
I left this part out before because INTR_STATE.done is more an implementation detail of the interrupt handling framework; I'd therefore remove the first part of the sentence and leave only the "mark the operation as completed".

No need to argue over that now, let's see if we have text that we can link to in the operation section in a follow-up.

Reply:
Ok, that makes sense. I feel like we need to say something about the `done` interrupt and `STATUS.busy` *somewhere*, but maybe this isn't the best place. Maybe we can tidy that up as a followup.

Reply:
There is a bit of documentation around them in the reggen generated stuff. I think we could at least add a little more to explicitly state `done` fires when an error is seen and be explicit about how `STATUS.busy` behaves relative to `start` and `done`. No need to do that in this PR though.